Rename redemption_nonce to redemption_context (#105)
ietf-wg-privacypass/base-drafts: draft-ietf-privacypass-auth-scheme.md

Unless otherwise specified, this document encodes protocol messages in TLS
notation from {{!TLS13=RFC8446}}, Section 3.

This document uses the terms "Client", "Origin", "Issuer", "Issuance Protocol",
and "Token" as defined in {{?I-D.ietf-privacypass-architecture}}. It additionally
uses the following terms in more specific ways:

- Issuer key: Keying material that can be used with an issuance protocol
to create a signed token.

- Token challenge: A requirement for tokens sent from an origin to a client, using
the "WWW-Authenticate" HTTP header. This challenge is bound to a specific token

> issuer and issuance protocol, and may be additionally bound to an origin name or
> origin context.

Reviewer: ```suggestion
issuer and issuance protocol, and may be additionally bound to a specific context or origin name.
```

hostname that is used to identify the issuer that is allowed to issue
tokens that can be redeemed by this origin.

- "redemption_context" is an optional field. If present, it allows the origin
to require that clients fetch tokens bound to a specific context, as opposed

> to reusing tokens that were fetched for other contexts. When present, this valid

Reviewer: ```suggestion
to reusing tokens that were fetched for other contexts. When present, this value
```

> is a fresh 32-byte context generated by the origin. This value can be a unique

Reviewer: ```suggestion
is a 32-byte context generated by the origin. This value can be a unique
```

It doesn't need to be "fresh" since one can compute it deterministically.
